src/functions/creating-functions.js

```diff
@@ -8,6 +8,10 @@
 //
 // TODO: write code below
 
+function increment(num) {
+  return num+=1;
```

Comment:
There is a shorthand way of adding and subtracting. 

Adding: 
```js
num++
```

Subtracting:
```js
num--
```